babel/src/test/resources/sql/spark.iq

```diff
@@ -448,4 +448,177 @@ EXPR$0
 [2.111111111111112, 2.1]
 !ok
 
+# Tests for CALCITE-6680
+# RexImpTable erroneously declares NullPolicy.NONE for IS_EMPTY
+# Added as Babel tests since they need the ARRAY() and MAP() functions
+
+CREATE OR REPLACE TABLE COMPLEX(
+   KEY INT, "list" INT ARRAY, "long" BIGINT, "map" MAP<VARCHAR, INT>, "set" INT MULTISET);
+(0 rows modified)
+
+!update
+
+INSERT INTO COMPLEX (VALUES
+ (0, NULL, 5, NULL, NULL),
```

Comment:
Maybe we can add unit tests like 'list is null' or 'map is null'?

Reply:
Added in a new commit, please check.